Modules/bookSet.js

```diff
@@ -0,0 +1,78 @@
+const bookSet = () => {
+  let books = JSON.parse(localStorage.getItem('bookData')) || [];
+
+  const addButton = document.querySelector('#addBook');
+  const formTitle = document.getElementById('title');
+  const formAuthor = document.getElementById('author');
+  const bookList = document.querySelector('#list');
+
+  class Book {
+    constructor() {
+      this.title = document.getElementById('title');
+      this.author = document.getElementById('author');
+    }
+
+    removeBook(id) {
+      let bookData = JSON.parse(localStorage.getItem('bookData'));
+      bookData = bookData.filter((local) => local.id !== parseInt(id, 10));
+      books = bookData;
+      localStorage.setItem('bookData', JSON.stringify(books));
+      this.displaylist();
+    }
+
+    displaylist() {
+      books.forEach((book) => {
+        const bookCard = document.createElement('li');
+        const removeButton = document.createElement('button');
+        removeButton.classList.add('my-button-container');
+
+        bookCard.innerHTML += `    
+      <p class="my-title-container">'${book.title}' by ${book.author}</p>
+      `;
+        bookCard.id = `data-${book.id}`;
+        removeButton.textContent = 'Remove';
+        removeButton.dataset.id = book.id;
+        removeButton.addEventListener('click', (e) => {
+          const { id } = e.target.dataset;
+          this.removeBook(id);
+          const bookEl = document.getElementById(`data-${book.id}`);
+          bookList.removeChild(bookEl);
+          window.location.reload();
+        });
+        bookCard.appendChild(removeButton);
+        bookList.appendChild(bookCard);
+        bookCard.classList.add('book-stack');
+      });
+    }
+
+    addBook(title, author) {
+      if (title !== '' && author !== '') {
+        const book = {
+          id: books.length + 1,
+          title,
+          author,
+        };
+
+        books.push(book);
+        localStorage.setItem('bookData', JSON.stringify(books));
+        this.displaylist();
+      }
+    }
+  }
+
+  const newBook = new Book();
+  newBook.displaylist();
```

Comment:
- [ OPTIONAL] Please do not instantiate the **Book** class in the module where it is defined. modules are meant to be imported and instantiated in the file where they are imported 👍🏾